cmd/send-email/main.go

```diff
@@ -13,6 +14,7 @@ var args struct {
     To []string `help:"Email recipents." required:""`
     Subject string `help:"Email subject." required:""`
     Body string `help:"Email body." required:""`
+    Course string `help:"Course ID." default:""`
```

Comment:
Add info in the help about how this is only necessary if there are user references that need to be resolved.